File: Examples/CSharp/GroupDocs.AssemblyExamples/ColumnNameExtractingDocumentTableLoadHandler.cs
using GroupDocs.Assembly.Data;

namespace GroupDocs.AssemblyExamples
{
    public class ColumnNameExtractingDocumentTableLoadHandler : IDocumentTableLoadHandler
    {
        public void Handle(DocumentTableLoadArgs args)
        {
            args.Options = new DocumentTableOptions { FirstRowContainsColumnNames = true };
        }
    }
}

File: Examples/VisualBasic/GroupDocs.Assembly.Examples/ColumnNameExtractingDocumentTableLoadHandler.vb

Imports GroupDocs.Assembly.Data

Public Class ColumnNameExtractingDocumentTableLoadHandler
    Implements IDocumentTableLoadHandler

    Public Sub Handle(args As DocumentTableLoadArgs) Implements IDocumentTableLoadHandler.Handle
        args.Options = New DocumentTableOptions()
        args.Options.FirstRowContainsColumnNames = True
    End Sub
End Class
